### PR TITLE
support serialize undefined

### PR DESCRIPTION
**Pull request by nqdy666**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by okuryu** [comment]
Hmm, could you add tests?

File: index.js [hunk 1]
```diff
@@ -134,15 +146,15 @@ module.exports = function serialize(obj, options) {
     if (typeof str !== 'string') {
         return String(str);
     }
-
+
```
Comment (okuryu) [inline index.js]: nit: I think the indent is unnecessary.

**Comment by okuryu** [comment]
published `serialize-javascript@1.9.0`. Thanks!

**Comment by src-code** [comment]
I believe this change introduced a regression, will open an issue

**Comment by src-code** [comment]
https://github.com/yahoo/serialize-javascript/issues/57

**Pull request merged**